pkg/filters/options.go

```diff
@@ -0,0 +1,147 @@
+package filters
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"os"
+	"sync"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/kubernetes"
+)
+
+// Options represents the flags and options for filtering unused Kubernetes resources, such as pods, services, or configmaps.
+// A resource is considered unused if it meets the following conditions:
+//   - Its age (measured from the last modified time) is within the range specified by older-than and newer-than flags.
+//     If older-than or newer-than is zero, no age limit is applied.
+//     If both flags are set, an error is returned.
+//   - Its size (measured in bytes) is within the range specified by MinSize and MaxSize flags.
+//     If MinSize or MaxSize is zero, no size limit is applied.
+//   - It does not have any labels that match the ExcludeLabels flag. The ExcludeLabels flag supports '=', '==', and '!=' operators,
+//     and multiple label pairs can be separated by commas. For example, -l key1=value1,key2!=value2.
+type Options struct {
+	// OlderThan is the minimum age of the resources to be considered unused
+	OlderThan string
+	// NewerThan is the maximum age of the resources to be considered unused
+	NewerThan string
+	// ExcludeLabels is a label selector to exclude resources with matching labels
+	// IncludeLabels conflicts with it, and when setting IncludeLabels, ExcludeLabels is ignored and set to empty
+	ExcludeLabels string
```

Comment:
nit: I think it would be more intuitive to call this something like `ExcludeSelector` to make it clearer it's a selector string.

Reply:
okay ， this good. I want to be consistent with the prompts in the command.  `cmd --exclude-labels` --> `cmd --exclude-selector` . (--selector keep similar parameters to kubectl)